test/dlc_tests/sanity/test_torchvision.py

```diff
@@ -12,7 +12,7 @@
 
 
 @pytest.mark.model("N/A")
-def test_torchvision_nms_training(pytorch_training):
+def test_torchvision_nms_training(pytorch_training, pt16_and_above_only):
```

Comment:
Why is this needed? Is the nms feature not available on the currently available PTv1.5.1 image?

Reply:
The feature being tested on this test seems to not have been added at the time.